exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if ((sentence = sentence.includes("code"))) {
```

Comment:
Good use of `includes` method.
`includes `is a predicate function, returning either true or false. 
Do you think that it needs to be assigned back to parameter sentence , on line 12? Generally, in an `IF` statement, we don't need to assign something to a variable

Reply:
Thanks for the tip...